app/scripts/model/dataProxy.js

```diff
@@ -678,32 +703,27 @@ window.DataManagerForIviz = (function($, _) {
                   }
                 });
 
-                _result.groups = {};
-                _result.groups.patient = {};
-                _result.groups.sample = {};
-                _result.groups.group_mapping = {};
-                _result.groups.patient.attr_meta =
-                  content.util
-                    .sortByClinicalPriority(_.values(_patientAttributes));
-                _result.groups.sample.attr_meta =
-                  content.util
-                    .sortByClinicalPriority(_.values(_sampleAttributes));
-                _result.groups.patient.data = _patientData;
-                _result.groups.patient.hasAttrData = _hasPatientAttrData;
-                _result.groups.sample.data = _sampleData;
-                _result.groups.sample.hasAttrData = _hasSampleAttrData;
-                _result.groups.patient.data_indices = {};
-                _result.groups.sample.data_indices = {};
-                _result.groups.patient.data_indices.patient_id =
-                  _patientDataIndicesObj;
-                _result.groups.sample.data_indices.sample_id =
-                  _sampleDataIndicesObj;
-                _result.groups.group_mapping.sample = {};
-                _result.groups.group_mapping.patient = {};
-                _result.groups.group_mapping.sample.patient =
-                  _samplesToPatientMap;
-                _result.groups.group_mapping.patient.sample =
-                  _patientToSampleMap;
+                _result.groups = {
```

Comment:
Could you please log this data structure in the wiki page? https://github.com/cBioPortal/iViz/wiki/Input-Data-JSON-Structure